include/irr/video/SubAllocatedDataBuffer.h

```diff
@@ -153,7 +153,7 @@ class SubAllocatedDataBuffer : public virtual core::IReferenceCounted
             auto allocation = mAllocator.getCurrentBufferAllocation();
 
             IGPUBuffer* retval;
-            static_if<is_std_get_0_defined<decltype(allocation)>::value >([&](auto f){
+            static_if<true/*is_std_get_0_defined<decltype(allocation)>::value*/ >([&](auto f){ // <-- todo, apparently MSVC SFINAE bug, however i can't even reproduce this in simpler form
```

Comment:
that will break, badly I have need for both in the current codebase

Also what MSVC version are you using? this compiles for us on windows.

Reply:
VS 2017 (15.9.3)
i'll try changing toolset version.
Also i'll have to update cmake because it generates projects for VS 2015 and maybe there's the differenece between our setup

Reply:
yea, that was it! With v141 toolset it builds just fine